web-bundle/src/main/java/com/graphhopper/http/GraphHopperManaged.java

```diff
@@ -72,7 +72,7 @@ public GraphHopperManaged(GraphHopperConfig configuration, ObjectMapper objectMa
             logger.error("Problem while reading border map GeoJSON. Skipping this.", e1);
             landmarkSplittingFeatureCollection = null;
         }
-        if (!configuration.getString("gtfs.file", "").isEmpty()) {
+        if (configuration.has("gtfs.file")) {
```

Comment:
could is still also be an empty string?  Or does .has() cover that case.

Reply:
yes, has() returns true even with an empty string, it basically just answers “does gtfs.file exist as a field in the yaml?” The issue here is that we have some steps (gtfs_links, export) where we want a GTFS graphhopper object, but we don’t provide a gtfs file in the command-line. This means gtfs.file exists but is "", empty, so the logic here would fall back to use an OSM graphhopper

Now, the decision is made purely on whether or not gtfs.file is in the config, vs. whether or not we pass in a gtfs file in the command line